lib/launch-fastify.js

```diff
@@ -114,6 +114,14 @@ async function launchFastify(file, options, address) {
       [openApiSpecification]: swaggerDefinition(serviceModule.swaggerDefinition, openApiSpecification),
       exposeRoute: true,
       transform: serviceModule.transformSchemaForSwagger,
+      refResolver: {
```

Comment:
IMO, you should give the ability to override this refResolver with the mergedOptions

Reply:
Do you mean something like
```
const refResolver = mergedOptions.refResolver || {
    buildLocalReference(json, baseUri, fragment, i) {
        ...
    }
}
```
?

Reply:
Exactly!

Reply:
It should be better `mergedOptions.swaggerRefResolver`, or `mergedOptions.oasRefResolver`

Reply:
I think it should be better to use `oasRefResolver`